### ISSUE TITLE
Schema URL versus URI

### ISSUE THREAD

**Issue by marcalff** [open-telemetry/opentelemetry-specification#2899]
The specification for traces indicates that the `TraceProvider` accepts a `schema_url` parameter, in the API to get a tracer.

```
[since 1.4.0] schema_url (optional): Specifies the Schema URL that should be recorded in the emitted telemetry.
```

This `schema_url` is then used in various places, when emitting a trace span.

Please clarify:

- is a **URL** actually intended here, meaning the URL should point to a document that actually exists ? In which case, what kind of document is expected to be hosted at this URL, using which format, and what process is expected to fetch the document from the said URL ?
- or is a **URI** only sufficient.

**Comment by carlosalberto** [comment]
Indeed, it seems we don't have any further explanation on this item, so I will mark it as accepted, and will assign it to @tigrannajaryan for a start.
